source/NoteWindow.cpp

```diff
@@ -84,6 +84,7 @@ NoteWindow::NoteWindow(int32 id)
 	entry_ref	*directory = NULL;
 	BRefFilter	*refFilter = NULL;
 	BString 	title("Untitled Note ");
+	bool unsaved = false;
```

Comment:
Move it into the constructor initializer list.

Reply:
Can you format your code based on [Haiku Coding Guidelines](https://www.haiku-os.org/development/coding-guidelines/)?